### ISSUE TITLE
Error while installing node.js

### ISSUE THREAD

**Issue by dsstutts** [os-js/OS.js#461]
An unexpected error occured, maybe a bug.

> Please report this if you think this is a bug. Include a brief description on how the error occured, and if you can; how to replicate it

```
ApplicationArduinoPackageManager.prototype.callOpkg/<@http://linino1.local/packages/target/ArduinoPackageManager/combined.js?23340:288:13
ApplicationArduinoPackageManager.prototype.callAPI/<@http://linino1.local/packages/target/ArduinoPackageManager/combined.js?23340:260:14
doAPICall/<@http://linino1.local/osjs.js:1756:7
_Handler.prototype.__callXHR/data.onerror@http://linino1.local/osjs.js:9466:9
onReadyStateChange@http://linino1.local/osjs.js:1266:11

```

**Comment by andersevenrud** [comment]
This was fixed in 502ba745c9717e662fe82c4a451e6286dd8bb3ec and will be forwarded to the Arduino team ASAP.

Thanks for reporting!

**Comment by andersevenrud** [comment]
PR: https://github.com/arduino-org/Arduino-OS/pull/23
